Resources/Prototypes/_NF/Recipes/Cooking/medical_recipes.yml

```diff
@@ -1,59 +1,59 @@
-  #pen recycling recipes
+  #pen recycling recipes #I think this is going to become cumbersome to maintain over time and the expanded reclaimer code makes it redundant.
 
-- type: microwaveMealRecipe
-  id: RecipeCleanEmergencyMedipen
-  name: sterilize and reset pen
-  result: BlankMediPen
-  time: 10
-  solids:
-    EmergencyMedipen: 1
-  reagents:
-    SpaceCleaner: 1
-  recipeType: MedicalAssembler # Frontier
+#- type: microwaveMealRecipe
+#  id: RecipeCleanEmergencyMedipen
+#  name: sterilize and reset pen
+#  result: BlankMediPen
+#  time: 5
+#  solids:
+#    EmergencyMedipen: 1
+#  reagents:
+#    SpaceCleaner: 1
+#  recipeType: MedicalAssembler # Frontier
 
-- type: microwaveMealRecipe
-  id: RecipeCleanAntiPoisonMedipen
-  name: sterilize and reset pen
-  result: BlankMediPen
-  time: 10
-  solids:
-    AntiPoisonMedipen: 1
-  reagents:
-    SpaceCleaner: 5
-  recipeType: MedicalAssembler # Frontier
+#- type: microwaveMealRecipe
+#  id: RecipeCleanAntiPoisonMedipen
+#  name: sterilize and reset pen
+#  result: BlankMediPen
+#  time: 5
+#  solids:
+#    AntiPoisonMedipen: 1
+#  reagents:
+#    SpaceCleaner: 5
+#  recipeType: MedicalAssembler # Frontier
 
-- type: microwaveMealRecipe
-  id: RecipeCleanBruteAutoInjector
-  name: sterilize and reset pen
-  result: BlankMediPen
-  time: 10
-  solids:
-    BruteAutoInjector: 1
-  reagents:
-    SpaceCleaner: 5
-  recipeType: MedicalAssembler # Frontier
+#- type: microwaveMealRecipe
+#  id: RecipeCleanBruteAutoInjector
+#  name: sterilize and reset pen
+#  result: BlankMediPen
+#  time: 5
+#  solids:
+#    BruteAutoInjector: 1
+#  reagents:
+#    SpaceCleaner: 5
+#  recipeType: MedicalAssembler # Frontier
 
-- type: microwaveMealRecipe
-  id: RecipeCleanBurnAutoInjector
-  name: sterilize and reset pen
-  result: BlankMediPen
-  time: 10
-  solids:
-    BurnAutoInjector: 1
-  reagents:
-    SpaceCleaner: 5
-  recipeType: MedicalAssembler # Frontier
+#- type: microwaveMealRecipe
+#  id: RecipeCleanBurnAutoInjector
+#  name: sterilize and reset pen
+#  result: BlankMediPen
+#  time: 5
+#  solids:
+#    BurnAutoInjector: 1
+#  reagents:
+#    SpaceCleaner: 5
+#  recipeType: MedicalAssembler # Frontier
 
-- type: microwaveMealRecipe
-  id: RecipeCleanRadAutoInjector
-  name: sterilize and reset pen
-  result: BlankMediPen
-  time: 10
-  solids:
-    RadAutoInjector: 1
-  reagents:
-    SpaceCleaner: 5
-  recipeType: MedicalAssembler # Frontier
+#- type: microwaveMealRecipe
+#  id: RecipeCleanRadAutoInjector
+#  name: sterilize and reset pen
+#  result: BlankMediPen
+#  time: 5
+#  solids:
+#    RadAutoInjector: 1
+#  reagents:
+#    SpaceCleaner: 5
+#  recipeType: MedicalAssembler # Frontier
```

Comment:
I'd prefer to remove these entirely vs. commenting them out.

If they handled being sprayed with cleaner and turned into blank medipens, that could work.  Could work like goblins and felinids being sprayed with water or space cleaner.